examples/jupyter/demo010_advection.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
ugh, I like these demos, but they are unplesant to review in the web view. I'll take a look from a local copy later

Reply:
How about the jupyter notebook. Can you review it in the web view now?

Reply:
It's still messy in the PR web view, which is just how GitHub works. I'll take a look at it on my computer on Monday